module-4/backend/week_one.md

```diff
@@ -1,19 +1,39 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+> The most important thing I learned was the foundational idea/definition of what an object is in js and that everything is basically an object with properties and functions
+
 2. What are some tools to help debug JavaScript code?
+> pryjs, debugger, developer console : elements, console, sources, and nework panes
+
 3. What are some tools you need in order to unit test your JavaScript?
+> (npm, )Mocha, Chai
+
 4. What is the syntax for invoking a function? Give an example.
+> funcionName(arg1, arg2, ...)
+
 5. What's `this` in JavaScript?
+> `this` typically refers to the object which contains or surrounds the current object
+
 6. What is Webpack and why is it useful?
+> Webpack minifies and concatenates our javascript files in to one output file for systems to more easily process
```

Comment:
Yep! JS is evolving faster than the browsers can keep up with, so build tools like Webpack help us make sure all our code is compatible with the browsers it's running on.